### ISSUE TITLE
3sec gif before showing roll result

### ISSUE THREAD

**Issue by mesqueeb** [ArtemGr/Sidekick#284]
I'd love to see a 3second GIF of dice being rolled and then that text to be edited and replaced with the roll result.

That way any time someone rolls, it's like we're all watching the dice. Must nicer for the atmosphere for remote DnD sessions!

What do you think @ArtemGr ?

**Comment by ArtemGr** [comment]
Like, generating an animated GIF? Sounds like an interesting option! You want to work on it?

**Comment by mesqueeb** [comment]
@ArtemGr I read somewhere you write Rust? I'm only proficient in JS and TS. : O
Otherwise, yeah I'd love to help out! I have a clear idea and I think it shouldn't be too hard!
Do you wanna chat on Discord? I'm `Luca Ban [Mesqueeb]#4774`

**Comment by ArtemGr** [comment]
I'm struggling with websocket implementation at #232. I mean, it's doable, but it will take more time and money that I'd like to. And having a JavaScript engine around might be very useful for #160. So I'm thinking... why not turn things upside down and switch the Discord websocket client to NodeJS?

**Comment by mesqueeb** [comment]
Yeah I think with nodeJS it's easier for people to contribute to the source code! I believe more ppl can help with JavaScript than Rust. But I'm sure Rust is great fun to write. 

It's a trade-off: if you appreciate code contributions and PRs I'd go for nodeJS! If you simply like Rust more and don't care about PRs, I'd stick to Rust! : )

Looking forward to see which direction you take this package.
